vpr/src/route/bucket.cpp

```diff
@@ -4,6 +4,63 @@
 #include "vtr_log.h"
 #include "vpr_error.h"
 
+/* Bucket spacing algorithm:
+ *
+ * The size in cost each bucket consumes is a fixed width determined by
+ * conv_factor_.  The bucket index equation is simply:
+ *
+ *  bucket index = cost * conv_factor_
+ *
+ * The default conv_factor_ is 1e12, e.g. each bucket is 1 picosecond wide.
+ *
+ * There two reasons to change conv_factor_:
+ *  - The maximum cost item in the bucket would require too many buckets in
+ *    heap_, and would cause memory usage to climb higher than desired.
+ *  - The front bucket contains too many items, making the pop operation too
+ *    cost insenstive.
+ *
+ * The other consideration is to avoid rescaling the buckets too often, as
+ * that operation consumes time without delivering useful work.
+ *
+ * To prevent rescaling constantly, the bucket heap determines if a rescaling
+ * is needed based on two conditions:
+ *
+ *  - The maximum item cost (max_cost_) would require a bucket index that is
+ *    greater than max_buckets_.  When this occurs, a rescaling is done to
```

Comment:
don't you rescale in this case to make the width of the buckets *larger* (not smaller)?

Reply:
Yep, you are right.  Fixed.